app/tests/spec/lib/xhr.js

```diff
@@ -12,7 +11,7 @@ define([
   'lib/xhr',
   'lib/promise'
 ],
-function (chai, sinon, $, _, Xhr, p, undefined) {
+function (chai, sinon, $, _, Xhr, p, undefined) { //eslint-disable-line no-shadow-restricted-names
 
```

Comment:
This convention of declaring `undefined` seems unnecessary. Let's just ditch it.